eo-runtime/pom.xml

```diff
@@ -95,9 +95,13 @@ SOFTWARE.
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <argLine>@{argLine} -Xss${stack-size} -Duser.language=ru -Duser.region=RU</argLine>
+          <argLine>@{argLine} -Xss${stack-size} -Duser.language=ru
+            -Duser.region=RU
```

Comment:
@volodya-lombrozo also here, looks a bit wrong to me. Why not all in one line?

Reply:
@yegor256 fixed